chornovil.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Viacheslav Chornovil</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <header>
+      <div class="part">
+        <p>I'm <b>Victoria</b> Vizniuk It's <span>bioghaphy.</span></p>
+      </div>
+      <div class="part">
+        <a href="">Home</a>
+        <a href="">CV</a>
+        <a href="">Blog</a>
+        <a href="">Contact</a>
+        <a href="">Log in</a>
+      </div>
+    </header>
+    <main>
+      <section>
+        <div class="text1">
+          <p>
+            <strong>Viacheslav Maksymovych Chornovil</strong>
+            <i
+              >(Ukrainian: В'ячесла́в Макси́мович Чорнові́л; 24 December 1937 – 25
+              March 1999)</i
+            >
+            was a <b>Ukrainian</b> politician and Soviet <b>dissident</b>.
+          </p>
+        </div>
+        <img src="images/3чорновіл.jpeg" alt="portret" />
+      </section>
+      <div class="greyback"></div>
+      <div class="main-title">
+        <div class="yellowback"></div>
+        <p><b>Viacheslav</b> Chornovil</p>
+      </div>
+      <div class="vstup">
+        <p>
+          As a prominent Ukrainian dissident in the Soviet Union, he was
+          arrested multiple times in the 1960s, 1970s, and 1980s for his
+          political views. From 1992 onwards, Chornovil was one of the leaders
+          of Rukh, the People's Movement of Ukraine, which was the first
+          opposition party in democratic Ukraine, and editor-in-chief of the
+          newspaper Chas-Time (Chas) from 1995. One of the most prominent
+          political figures of the 1980s and 1990s, Chornovil paved the way for
+          contemporary Ukraine to regain its independence.
+        </p>
+      </div>
+      <div class="nav">
+        <p>
+          <span>Early <span class="grey">life</span> and education</span>
+          <span class="black">Journalist</span>
+        </p>
+        <p>
+          <span class="black">Dissident</span>
+          <span>Politician</span>
+        </p>
+        <p>
+          <span>Opposition to Leonid <span class="grey">Kuchma</span></span>
+        </p>
+        <p>
+          <span class="black">Death and remembrance</span>
+        </p>
+      </div>
+      <article>
+        <div class="picture1">
```

Comment:
Lets dont use numbers in the end of class or vars.
We could rename it as chornovil_picture..